model/team_member.go

```diff
@@ -44,6 +44,13 @@ type EmailInviteWithError struct {
 	Error *AppError `json:"error"`
 }
 
+type TeamMembersGetOptions struct {
+	// Sorting option
```

Comment:
Can we document what values we accept here?

Reply:
Done! Thanks